impeller/docs/benchmarks.md

```diff
@@ -0,0 +1,22 @@
+# Impeller Benchmarks
```

Comment:
We should link to this doc [from the main Impeller readme](https://github.com/flutter/engine/blob/main/impeller/README.md#documentation-references-and-additional-reading).